utils/tests/utils.go

```diff
@@ -130,6 +141,21 @@ func compare(expected, actual []string) error {
 	return nil
 }
 
+// Actual list contains dir and files, so we check prefix (for dirs) or equality (for files)
+func isExclusivelyExistLocally(expected, actual []string) error {
+	for _, v := range actual {
+		for i, r := range expected {
+			if strings.HasPrefix(r, v) || v == r {
+				break
+			}
+			if i == len(actual)-1 {
+				return errors.New("Should not have : " + v)
```

Comment:
Following our discussion -
Add t parameter to isExclusivelyExistLocally and add t.Error. Don't return err.
```suggestion
				t.Error("Should not have : " + v)
```